MastodonSDK/Sources/MastodonUI/View/Content/UserView+ViewModel.swift

```diff
@@ -22,10 +26,17 @@ extension UserView {
         @Published public var authorAvatarImageURL: URL?
         @Published public var authorName: MetaContent?
         @Published public var authorUsername: String?
+        @Published public var authorFollowers: Int?
+        @Published public var authorVerifiedLink: String?
+        @Published public var user: MastodonUser?
     }
 }
 
 extension UserView.ViewModel {
+    private static var metricFormatter: MastodonMetricFormatter = {
+        return MastodonMetricFormatter()
+    }()
```

Comment:
Is there are a reason that this is a computed property and not just something like 

```suggestion
    private static var metricFormatter = MastodonMetricFormatter()
```

? Or is it merely to match the coding-style?

Reply:
Nope, no real reason, maybe I did some testing with this before actually using / implementing `MastodonMetricFormatter`, I'm happy to change this 👍